### PR TITLE
Change grpc- prefix to proto-.

### PR DESCRIPTION
**Pull request by lukesneeringer**
We are changing the `grpc-` prefix for Python packages to `proto-`. This PR performs the necessary aspect of this change in artman.

It must be co-merged alongside:

  * googleapis/googleapis#239
  * googleapis/toolkit#930

### CODE REVIEW

File: pipeline/tasks/package_metadata_tasks.py [hunk 1]
```diff
@@ -28,6 +28,8 @@ class PackageMetadataConfigGenTask(task_base.TaskBase):
     def execute(self, api_name, api_version, organization_name, output_dir,
                 package_dependencies_yaml, package_defaults_yaml, repo_root,
                 src_proto_path):
+        print 'generating package metadata'
```
Comment (geigerj) [inline pipeline/tasks/package_metadata_tasks.py]: remove debug line?

File: pipeline/pipelines/gapic_generation_pipeline.py [hunk 2]
```diff
@@ -137,6 +138,7 @@ def _get_gapic_package_tasks(self, **kwargs):
 def get_gapic_task_factory(language):
     cls = _GAPIC_TASK_FACTORY_DICT.get(language)
     if cls:
+        print cls()._get_gapic_codegen_tasks(language='python')
```
Comment (geigerj) [inline pipeline/pipelines/gapic_generation_pipeline.py]: remove debug line?

File: pipeline/pipelines/gapic_generation_pipeline.py [hunk 3]
```diff
@@ -104,9 +104,10 @@ def get_invalid_kwargs(self):
 class _PythonGapicTaskFactory(GapicTaskFactoryBase):
 
     def _get_gapic_package_tasks(self, **kwargs):
-        return [gapic_tasks.GapicCleanTask,
-                gapic_tasks.GapicCopyTask,
-                gapic_tasks.GapicPackmanTask]
+        return [
+            gapic_tasks.GapicCleanTask,
+            gapic_tasks.GapicCopyTask,
```
Comment (geigerj) [inline pipeline/pipelines/gapic_generation_pipeline.py]: Worth filing an issue to make this formatting consistent, and enforce it with Travis?

**Comment by lukesneeringer** [comment]
This should be good to go; I had a couple `flake8` issues which are fixed. Additionally, `flake8` was being specified as a dep in `test-requirements.txt` and `tox.ini`; I changed it to the former only.

**Comment by geigerj** [comment]
@lukesneeringer Could you check why the Travis build is failing? It looks like one of the tests is failing.

File: tox.ini [hunk 4]
```diff
@@ -11,12 +11,14 @@ skipdist = True
 setenv =
        PYTHONPATH = {toxinidir}
 
-deps = -r{toxinidir}/test-requirements.txt
+# The long URL is temporary; when protobuf can successfully install from egg again,
```
Comment (geigerj) [inline tox.ini]: There's a new version: https://pypi.python.org/pypi/protobuf/3.2.0rc1.post1

If that works, let's remove this.

**Pull request merged**